### PR TITLE
remove common project code

### PR DESCRIPTION
**Pull request by wy65701436**
Signed-off-by: Wang Yan <wangyan@vmware.com>

### CODE REVIEW

**Review by heww — APPROVED** [review]

File: src/pkg/project/models/project.go [hunk 1]
```diff
@@ -15,14 +15,192 @@
...
+func (p *Project) AutoScan() bool {
+	auto, exist := p.GetMetadata(ProMetaAutoScan)
+	if !exist {
+		return false
+	}
+	return isTrue(auto)
+}
+
+// FilterByPublic returns orm.QuerySeter with public filter
+func (p *Project) FilterByPublic(ctx context.Context, qs orm.QuerySeter, key string, value interface{}) orm.QuerySeter {
```
Comment (reasonerjt) [inline src/pkg/project/models/project.go:137]: Is this being used? It doesn't look correct under the structure we agreed on.

Comment (wy65701436) [reply src/pkg/project/models/project.go:137]: cc @heww

Comment (reasonerjt) [reply src/pkg/project/models/project.go:137]: I didn't know this was based on convention.  
I'm fine with this approach.

File: src/pkg/project/dao/dao.go [hunk 2]
```diff
@@ -70,7 +70,7 @@ func (d *dao) Create(ctx context.Context, project *models.Project) (int64, error
 			return orm.WrapConflictError(err, "The project named %s already exists", project.Name)
 		}
 
-		member := &Member{
+		member := &models.Member{
```
Comment (ywk253100) [inline src/pkg/project/dao/dao.go:73]: There is another `Member` definition under package `pkg/member`, seems they are duplicated?

Comment (wy65701436) [reply src/pkg/project/dao/dao.go:73]: will raise another PR to handle this.

**Pull request merged**